src/styles/blocks/container.scss

```
@@ -0,0 +1,38 @@
+@import "./mixin.scss";
+@import "./variables.scss";
+
+.container {
+  display: flex;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(- 50%, - 50%);
+  justify-content: center;
+  width: 600px;
+  height: 600px;
+  margin: auto;
+  border: 1px solid $c-container;
+  border-radius: 50%;
+  background-color: $c-container;
+
+  &__minutes {
+    @include strilka(33%, $c-1, 2,
+    3600s, 1);
+  }
+
+  &__seconds {
+    @include strilka(48%, $c-2, 1,
+    60s, 60);
+  }
+
+  &__circle {
+    position: absolute;
+    bottom: 50%;
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+    z-index: 3;
+    transform: translateY( 50%);
+    background-color: $c-1;
```

not clearly what does this name means?) O_o
```suggestion
    background-color: $center-color;
    background-color: $main-color;
```